src/test/moves/ceaseless_edge.test.ts

```diff
@@ -12,6 +12,7 @@ import { Species } from "#enums/species";
 import { ArenaTagType } from "#app/enums/arena-tag-type.js";
 import { allMoves } from "#app/data/move.js";
 import { ArenaTagSide, ArenaTrapTag } from "#app/data/arena-tag.js";
+import { Abilities } from "#app/enums/abilities.js";
```

Comment:
```suggestion
import { Abilities } from "#app/enums/abilities";
```

Idk who even started that `.js` shenanigan when we are writing in TS

Reply:
I didn't even notice this, I just used the auto-import suggestion from VSCode. Fixed in latest commit